### DISCUSSION TITLE
Change Default Style

### DISCUSSION THREAD

**Discussion by CodeCup7** [palexdev/MaterialFX#294] [Q&A]
Hi :)

I know that I can add my own style to each control. Unfortunately, with the TableView, I can't get to the style of the filter dialog.
Wouldn't it be easier if the STYLESHEET field wasn't final? Then you could replace the style of the control with your own (e.g. dark mode).

**Comment by palexdev** [comment]
I'm not sure if it would work as that stylesheet is passed to an overridden method that is automatically called by JavaFX at some unspecified point in time
It's strange that you are not able to customize it though, maybe create an issue and share a MRE so that I can reproduce it and find a solution

Although it's very early to speak about it, I'm working on a rewrite of MaterialFX (rewrite branch yon can check the work), CSS management will be much easier and much more reliable
But as I said it's way too soon to speak about it I'm still implementing the buttons lol

**Comment by CodeCup7** [comment]
It's a great job overall :) 

When I add a TableView control, I can style it with CSS. But there is no way I know of to use styles on the dialog box that pops up when the filter is called. This is a new stage, not a popup date :)

**Comment by stefanofornari** [comment]
I had the same issue and the problem is that all classes use the userAgent style sheet that overrides everything. The only way I found to work around it was to create a new table skin and down to the relevant classes adding my own stylesheet. In it I have put the customization I needed. Use the same pattern for the classes that have the same problem.
